waffle-jest/test/setupJest.ts

```diff
@@ -0,0 +1,4 @@
+import {waffleJest} from '../src';
+
+jest.setTimeout(10000);
+expect.extend(waffleJest);
```

Comment:
It would be nice to have a bit of readme somewhere that specifies how to actually use the matchers in your waffle project

Reply:
That is a good idea. I just want to make sure this is something that the project maintainers want before writing it.